core/kernel/dt_driver_test.c

```diff
@@ -249,6 +258,43 @@ static TEE_Result dt_test_consumer_probe(const void *fdt, int node,
 		dt_test_state.probe_resets = SUCCESS;
 	}
 
+	if (IS_ENABLED(CFG_DRIVERS_GPIO)) {
+		struct gpio *gpio = NULL;
+
+		DT_TEST_MSG("Probe gpio controllers");
+
+		res = gpio_dt_get_by_index(fdt, node, 0, "foo", &gpio);
+		if (res)
+			goto err_probe;
+
+		if (gpio_get_direction(gpio) != GPIO_DIR_IN) {
+			EMSG("Unexpected gpio_get_direction return value");
```

Comment:
Please appen parentheses to all function labels printed in the trace messages:
			`EMSG("Unexpected gpio_get_direction() return value");`
here and below.